platform/ios/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ### Other changes
 
 * Xcode 8.0 or higher is now recommended for using this SDK. ([#8775](https://github.com/mapbox/mapbox-gl-native/pull/8775))
+* Fixed a crash when calling `MGLMultiPolygon.coordinate` [#8713](https://github.com/mapbox/mapbox-gl-native/pull/8713)
```

Comment:
We’ve already branched for iOS SDK v3.6.0: either retarget and rebase this PR on the release-ios-v3.6.0-android-v5.1.0 branch or add it to [this project](https://github.com/mapbox/mapbox-gl-native/projects/18) to be cherry-picked into that branch.